src/platform/silabs/multi-ota/OTAHooks.cpp

```diff
@@ -16,39 +16,55 @@
  *    limitations under the License.
  */
 
-#include <include/platform/CHIPDeviceLayer.h>
 #include <platform/silabs/multi-ota/OTAMultiImageProcessorImpl.h>
 
 #include <app/clusters/ota-requestor/OTARequestorInterface.h>
 
 #include <platform/silabs/multi-ota/OTAFactoryDataProcessor.h>
+
+#ifndef SLI_SI91X_MCU_INTERFACE
```

Comment:
Just to confirm. This is used for efr32 and with NCP but not with 917 soc correct?

Reply:
Correct `SLI_SI91X_MCU_INTERFACE` for 917 SoC